gst/droidcamsrc/gstdroidcamsrc.c

```diff
@@ -105,14 +105,15 @@ enum
 
 static guint droidcamsrc_signals[LAST_SIGNAL];
 
-#define DEFAULT_CAMERA_DEVICE          GST_DROIDCAMSRC_CAMERA_DEVICE_PRIMARY
+#define DEFAULT_CAMERA_DEVICE          0
```

Comment:
It would have been useful in the past to have had this configurable. The Gemini without the extra camera module only had the front-facing camera 1. It might be simpler now to drop this completely, and choose the default as the lowest numbered camera that is present.